python/core/is-your-python-healthy/python-debugger-ii.md

```diff
@@ -93,22 +93,16 @@ Finally, whenever you want to leave `pdb` console, type `quit` or `exit`.
 ---
 ## Practice
 
-What is the error this snippet will throw?
-```py
-name = input('What\'s your name?\n')
-def interaction()
-  print(name)
-  my_name = 'Seba'
-  if name is my_name:
-    print("We have the same name!")
-```
-???
+Match the command to what it does.
 
+The `???` command will **iterate** through the loops showing exactly what the **loop** is doing.
+The `???` command executes the next **statement**, without showing everything that happened to evaluate the call.
+The `???` command has **optional arguments** that must be two `int` values that define the range of lines the user wants to list.

```

Comment:
both of these use the method's name in their explanation. maybe we can find a different way of phrasing these without giving the answer in the question's text?